etc/preload-daemon

```diff
@@ -29,10 +29,10 @@ source "${DIRECTORY}/preload" "source"
 #variable 2 is 'once', if you only want this to run once and exit.
 
 #generate list of folders to preload, including special folders
-folders="$(read_category_files | awk -F '|' '{print $2}' | sort | uniq | grep .)
+folders="All Apps
+$(read_category_files | awk -F '|' '{print $2}' | sort | uniq | grep .)
 Installed
-Packages
-All Apps"
+Packages"
```

Comment:
I'd be interested to know your reasoning behind this change. This will preload `All Apps` first, to the detriment of all the others. It would seem logical to preload the smaller folders first.

Reply:
The logic was that a user may be more likely to click on All Apps than any other single category.

In my mind the most likely categories would go All Apps -> Installed -> everything else

You can change it back though, it doesn't really matter